gravitee-apim-rest-api/gravitee-apim-rest-api-services/gravitee-apim-rest-api-services-sync/src/main/java/io/gravitee/rest/api/services/sync/SyncManager.java

```diff
@@ -102,6 +128,12 @@ public void refresh() {
             log.error("An error occurs while synchronizing dictionaries", ex);
         }
 
+        try {
+            synchronizeLicenses(nextLastRefreshAt);
+        } catch (Exception ex) {
+            log.error("An error occurs while synchronizing licenses", ex);
+        }
```

Comment:
Just to stay consistent, I would have to move it before loading APIs so we are sure the licenses are loaded before anything else is synchronized.